app/components/shared/inputs/SliderInput.vue.ts

```diff
@@ -29,12 +30,14 @@ export default class SliderInput extends BaseInput<number, ISliderMetadata>  {
     // Hack to prevent transitions from messing up slider width
     setTimeout(() => {
       if (this.$refs.slider) this.$refs.slider.refresh();
+      this.isFullyMounted = true;
     }, 500);
   }
 
   @throttle(500)
   updateValue(value: number) {
-    this.$emit('input', this.roundNumber(value));
+    if (!this.isFullyMounted) return;
```

Comment:
so users cant interact with is the first 500ms after it loads?

Reply:
Yes, the reason I added this condition was that VueSlider triggered an `input` event after the refresh call in `mounted` hook above. Didn't have time to investigate why it is going. However, I can't imagine so crazy fast user that can switch tabs and slider for less than 500ms

Reply:
it seems a little iffy to me since its a decently long timeout but i guess it'll work for now until we find out exactly what's going on